ResearchKit.xcodeproj/project.pbxproj

```diff
@@ -1944,6 +1953,17 @@
 			name = Misc;
 			sourceTree = "<group>";
 		};
+		CB16A6EB1BB0D0660043CDD8 /* Wait Task */ = {
+			isa = PBXGroup;
```

Comment:
Refactor the group name to be `Wait Step` and move the group under the `Step` group. There are other types of steps located under there as well, such as `Instruction Step` or `Image Capture Step`